Clarify language about Smart Placement and RPC (#13903)
cloudflare/cloudflare-docs: content/workers/runtime-apis/rpc/_index.md

## Limitations

> - [Smart Placement](/workers/configuration/smart-placement/) is currently ignored when making RPC calls. If Smart Placement is enabled for Worker A, and Worker B declares a [Service Binding](/workers/runtime-apis/bindings) to it, when Worker B calls Worker A via RPC, Worker A will run locally, on the same machine.

Reviewer: ```suggestion
- [Smart Placement](/workers/configuration/smart-placement/) is currently ignored when making RPC calls. If Smart Placement is enabled for Worker A, and Worker B has a [Service binding](/workers/runtime-apis/bindings) configured that calls Workers A, when Worker B calls Worker A via RPC, Worker A will run locally, on the same machine.
```
Optional language suggestion as 'declares' is not a verb we usually use when referencing bindings.
